src/app/static/src/app/components/plots/table/Table.vue

```diff
@@ -11,17 +11,19 @@
                         @update:model-value="changeFilter"
                         :placeholder="translate('typeSearch')"></b-form-input>
                     <b-input-group-append>
-                        <b-button :disabled="!filter" @click="filter = ''" v-translate="'clearText'"></b-button>
+                        <b-button :disabled="!filter"
+                        @click="filter = ''"
+                        v-translate="'clearText'"
+                        size="sm"></b-button>
                     </b-input-group-append>
                 </b-input-group>
             </b-form-group>
             <b-table
                 striped hover
                 :fields="fields"
                 :items="filteredData"
-                :sort-by="sortBy"
-                :sort-desc="sortDesc"
```

Comment:
https://mrc-ide.myjetbrains.com/youtrack/issue/mrc-4325/Add-filtering-back-into-Area-Table-Indicators

So the thing is, it does work but you don't get the double arrow icon and the icons just appear when you click on the column headers so I thought it is best to remove it anyway since the tables will be getting a big overhaul